components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2627,6 +2628,36 @@ where
         self.fsm.hibernate_state.count_vote(from.get_id());
     }
 
+    fn on_trace_peer_availability_info(&mut self, from: &metapb::Peer, msg: &ExtraMessage) {
+        if !self.fsm.peer.is_leader() {
+            let mut resp = ExtraMessage::default();
+            resp.set_type(ExtraMessageType::MsgTracePeerAvailabilityInfo);
+            resp.wait_data = self.fsm.peer.wait_data;
+            self.fsm
+                .peer
+                .send_extra_message(resp, &mut self.ctx.trans, from);
+            info!(
+                "peer responses availability info to leader";
+                "region_id" => self.region().get_id(),
+                "peer_id" => self.fsm.peer.peer.get_id(),
+                "leader_id" => from.id,
+            );
```

Comment:
I'm wondering if info! is too much logs. Can it be debug?

Reply:
Large numbers changed to non-witness is possible, will change to debug.